src/core/mdn-annotation.js

```diff
@@ -31,6 +31,14 @@ const MDN_BROWSERS = {
   webview_android: "WebView Android",
 };
 
+const localizationStrings = {
+  en: {
+    inAllEngines: "This feature is in all current engines.",
```

Comment:
"major" engines, maybe. Servo is an engine too 😄

Reply:
Was copying Bikeshed

Reply:
Do we need to sync the messages?

Reply:
Nope. We need something for starters though..

Reply:
Agree... we should start with this set. It's nice that both ReSpec and BikeShed are showing the same boxes/info for now.

Reply:
I think adding "major" should still be fine, though. EdgeHTML is still in a lot of machines and there are also other minor ones.

Reply:
Is there a reason you don't want it? 😕

Reply:
Oops. Missed your newer comments before merge @saschanaz. Will improve messages in a followup

Reply:
"This feature is in all major engines"

Reply:
@saschanaz do you think this message is fine? Open to suggestions.

Reply:
That looks fine 😁